i18n.php

```diff
@@ -264,4 +264,21 @@
         'undefined_count'         => '{1,21,31} %count% не определен|]1,Inf] %count% не определено',
         'skipped_count'           => '{1,21,31} %count% пропущен|]1,Inf] %count% пропущено',
     ),
+    'cn'    => array(
```

Comment:
Isn’t it `zh`?

Reply:
`zh` will be nice ,I will fix it .

Reply:
I don’t know enough to say if it’s correct :)

You could consider contributing gherkin translations to cucumber/cucumber too